hubmap/frontend/cypress/integration/hubmap_spec.js

```diff
@@ -32,4 +32,32 @@ describe('HuBMAP', () => {
     // Footer:
     cy.contains('Supported by the NIH Common Fund');
   });
+
+  describe('The Login Page', function () {
+    before(() => {
+      cy.server();
+      const api = 'http://localhost:8000/';
+      cy.route(`${api}/api/?format=json`, 'fixture:base.json');
+      cy.route(`${api}/api/colors/?format=json`, 'fixture:colors.json');
+      cy.route(`${api}/api/genes/?format=json`, 'fixture:genes.json');
+      cy.route(`${api}/auth/login/globus`, 'fixture:login.json');
+      cy.setCookie('email', 'test@gmail.com')
+    });
+
+    it('sets auth cookie when logging in via form submission', function () {
+      // click login
+      cy.get('login').click()
+  
+      // we should be redirected to / (home)
+      cy.url().should('include', '/')
+
+      cy.get("loggedin-menu").should('exist')
+  
+      // our auth cookie should be present
+      cy.getCookie('email').should('exist')
+  
+      // UI should reflect this user being logged in
+      cy.get('loggedinemail').should('contain', 'test')
```

Comment:
I think the test could be more broad, and less fragile: test@gmail.com should just be somewhere on the page, right... we don't really care which element it's in?

Reply:
As I mentioned in one another comment, hubmap_spec.js is implemented by you, we can leave it with your version.

Reply:
@mccalluc can you pls take a look at this: https://github.com/hubmapconsortium/hubmap-data-portal/pull/103#discussion_r315265347